tests/jest/unit/reports/ReportCountSummary24h.spec.js

```diff
@@ -75,6 +77,7 @@ test('ReportCountSummary24h.timeRange', () => {
   });
 });
 
+/*
```

Comment:
can we modify this test in the same way as other tests?  we shouldn't just stop testing this one report type.